pages/menu.js

```diff
@@ -0,0 +1,254 @@
+import React from 'react'
+import MainLayout from '../layouts/mainLayout/MainLayout'
+import Image from 'next/image'
+import images from '../utils/images'
+
+import {
+    Container,
+    Box,
+    Button,
+    Typography as Font,
+    List,
+    ListItem,
+    ListItemText,
+    Divider,
+} from '@mui/material'
+// const DATABASE_URL = 'http://54.177.165.108:3000/api/items/list'
+const fakeData = {
```

Comment:
Add functionality for filtering out by category
expect categories from the backend to be an array
remember to solve any merge conflicts